downloads.html

```diff
@@ -6,12 +6,58 @@
 	<div class="right">
     <h1>Download</h1>
 
-    <p>3.0.0 is the latest release. The current stable version is 3.0.0.</p>
+    <p>3.1.0 is the latest release. The current stable version is 3.1.0.</p>
 
     <p>
     You can verify your download by following these <a href="https://www.apache.org/info/verification.html">procedures</a> and using these <a href="https://downloads.apache.org/kafka/KEYS">KEYS</a>.
     </p>
 
+    <span id="3.1.0"></span>
+    <h3 class="download-version">3.1.0<a href="#3.1.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+    <ul>
+        <li>
+            Released January 24, 2022
+        </li>
+        <li>
+            <a href="https://downloads.apache.org/kafka/3.1.0/RELEASE_NOTES.html">Release Notes</a>
+        </li>
+        <li>
+            Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/3.1.0/kafka-3.1.0-src.tgz">kafka-3.1.0-src.tgz</a> (<a href="https://downloads.apache.org/kafka/3.1.0/kafka-3.1.0-src.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/3.1.0/kafka-3.1.0-src.tgz.sha512">sha512</a>)
+        </li>
+        <li>
+            Binary downloads:
+            <ul>
+                <li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/3.1.0/kafka_2.12-3.1.0.tgz">kafka_2.12-3.1.0.tgz</a> (<a href="https://downloads.apache.org/kafka/3.1.0/kafka_2.12-3.1.0.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/3.1.0/kafka_2.12-3.1.0.tgz.sha512">sha512</a>)</li>
+                <li>Scala 2.13 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/3.1.0/kafka_2.13-3.1.0.tgz">kafka_2.13-3.1.0.tgz</a> (<a href="https://downloads.apache.org/kafka/3.1.0/kafka_2.13-3.1.0.tgz.asc">asc</a>, <a href="https://downloads.apache.org/kafka/3.1.0/kafka_2.13-3.1.0.tgz.sha512">sha512</a>)</li>
+            </ul>
+            We build for multiple versions of Scala. This only matters if you are using Scala and you want a version
+            built for the same Scala version you use. Otherwise any version should work (2.13 is recommended).
+        </li>
+    </ul>
+
+    <p>
+        Kafka 3.0.0 includes a number of significant new features. Here is a summary of some notable changes:
```

Comment:
Just noticed this should this be `3.1.0` here?

Reply:
Good catch, fixed.